tests/config.tests/apply.json

```diff
@@ -2,11 +2,10 @@
   "version": 1,
   "blobset": {
     "blobs": {
-      "codebase": {
-        "stage": "apply",
-        "target": "e21637e422d3153b866279af92ec1b1fd17c8846"
+      "hello_payload": {
+        "phase": "apply"
```

Comment:
I think the **apply** (and others) _phase_ might also have the _type/src/href/whatever_ fields defined as well, and if so, then it means "download and apply" -- sometimes you don't really care and there's no need to then implicitly walk through all the states.

Reply:
I think that we need a specific phase that allows one-shot mode, otherwise two problems can result:

1. In the prep (download) phase the device pulls content and saves it. If the urls etc change between prep and apply, either the device has to keep track of the former prep blob to see the difference, or it has to needlessly do the download again.

Reply:
2. It is not clear whether all devices will want to do the 3 step or 1 step if the lines are blurred between how they work